packages/website/components/account/filesManager/uploadsTable.js

```diff
@@ -146,68 +167,182 @@ const UploadsTable = ({ content, hidden, onFileUpload, onUpdatingChange, showChe
 
     onUpdatingChange(false);
     setSelectedUploads([]);
-
+    const [sortBy, sortOrder] = order.split(',');
     getUploads({ size, page, sortBy, sortOrder });
     setDeleteSingleCid('');
     deleteModalState[1](false);
     refetch();
   }, [
     onUpdatingChange,
     selectedUploads,
+    order,
     getUploads,
     size,
     page,
-    sortBy,
-    sortOrder,
     deleteModalState,
     refetch,
     deleteSingleCid,
     deleteUpload,
   ]);
 
+  /** Deletes a single uploads. */
   const onDeleteSingle = useCallback(
-    async cid => {
+    async (/** @type {Upload} */ upload) => {
       deleteModalState[1](true);
-      setDeleteSingleCid(cid);
+      setDeleteSingleCid(upload.cid);
     },
     [deleteModalState]
   );
 
-  const onEditToggle = useCallback(
-    targetCID => async (/** @type {string|undefined} */ newFileName) => {
-      setNameEditingId(targetCID !== nameEditingId ? targetCID : undefined);
-
-      const fileTarget = uploads.find(({ cid }) => cid === targetCID);
-      if (!!fileTarget && !!newFileName && newFileName !== fileTarget.name) {
-        onUpdatingChange(true);
-        await renameUpload(targetCID, newFileName);
-        fileTarget.name = newFileName;
-        onUpdatingChange(false);
-      }
-    },
-    [nameEditingId, uploads, onUpdatingChange, renameUpload]
-  );
-
+  /** Handles re-fetching upload date. */
   const refreshHandler = useCallback(() => {
+    const [sortBy, sortOrder] = order.split(',');
     getUploads({ size, page, sortBy, sortOrder });
     showCheckOverlay();
-  }, [getUploads, size, page, sortBy, sortOrder, showCheckOverlay]);
+  }, [getUploads, size, page, order, showCheckOverlay]);
```

Comment:
Why have we missed `sortBy`, `sortOrder` here?

Reply:
I've removed `sortBy` and `sortOrder` params in favour of a single combined `order` param, felt a little cleaner and stops the 2 params getting de-synced!